database/factories/LocationFactory.php

```diff
@@ -25,4 +25,9 @@ public function definition()
             'image' => rand(1, 9).'.jpg',
         ];
     }
+
+    public function deleted(): self
```

Comment:
I don't believe we use soft deletes on Locations, do we?

Reply:
<img width="638" alt="image" src="https://github.com/snipe/snipe-it/assets/7305753/e74e40d9-438d-4963-afe2-fc4f11f86648">
Looks like we do.

Reply:
Derp, you're right. I think we added that trait later, so the `deleted_at` was in a weird position in the schema (after `phone`).